frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -548,11 +548,14 @@ export async function getUsersWithActiveTasks(projectId: string): Promise<Array<
 
 // ### Private Links
 
-export function createPrivateLink(annotationId: string): Promise<ZarrPrivateLink> {
+export function createPrivateLink(
+  annotationId: string,
+  initialExpirationPeriodInDays: number = 30,
+): Promise<ZarrPrivateLink> {
   return Request.sendJSONReceiveJSON("/api/zarrPrivateLinks", {
     data: {
       annotation: annotationId,
-      expirationDateTime: null,
+      expirationDateTime: new Date().getTime() + initialExpirationPeriodInDays * 24 * 3600 * 1000,
```

Comment:
More readable, less error prone (`moment` might need to be imported):

```suggestion
      expirationDateTime: moment().add(initialExpirationPeriodInDays, "days").valueOf(),
```